frontend/src/views/c-authorship.vue

```diff
@@ -5,9 +5,9 @@
   .toolbar--multiline
     a(
       v-if="activeFilesCount < selectedFiles.length",
-      v-on:click="expandAll()"
+      v-on:click="toggleAllFileActiveProperty(true)"
     ) show all file details
-    a(v-if="activeFilesCount > 0", v-on:click="collapseAll()") hide all file details
+    a(v-if="activeFilesCount > 0", v-on:click="toggleAllFileActiveProperty(False)") hide all file details
```

Comment:
Should this be a lowercase `false`?

Reply:
Yes it should, thanks for pointing this out! Will fix this